src/Bicep.LangServer.IntegrationTests/Bicep.LangServer.IntegrationTests.csproj

```diff
@@ -24,4 +24,10 @@
     <ProjectReference Include="..\Bicep.LangServer\Bicep.LangServer.csproj" />
   </ItemGroup>
 
+	<ItemGroup>
+		<Content Include="Completions\SnippetTemplates\*.bicep">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</Content>
```

Comment:
The convention we've been following is to use `EmbeddedResource` instead of copying files to output - to avoid the flaky MSTest behavior with test artifects. We've also got some helper methods to make this easier

Here's an example of declaring in the csproj:
https://github.com/Azure/bicep/blob/a146127dbd0bbfdf628b70c843272b1a73096e20/src/Bicep.Core.Samples/Bicep.Core.Samples.csproj#L20-L21

Retrieving the names in a test:
https://github.com/Azure/bicep/blob/cdc013bdbff80422cee6253cdacb57327b7e6d1b/src/Bicep.Core.Samples/ExamplesTests.cs#L64-L86

Copying to the test output directory:
https://github.com/Azure/bicep/blob/cdc013bdbff80422cee6253cdacb57327b7e6d1b/src/Bicep.Core.Samples/ExamplesTests.cs#L116